src/themes/money/theme.json

```diff
@@ -138,102 +146,80 @@
   },
 
   "----------------------------- Variants": "-----------------------------",
-  "modules": {
-
-    "content-hero-module": {
-      "h1": {
-        "mb": "sm",
-        "mt": 0,
-        "pt": ["sm", "sm", "lg"]
-      },
-
-      "h4": { "fontSize": "md", "fontWeight": "normal" },
-
-      "p": {
-        "width": ["auto", "grid.md.5", "grid.lg.5"],
-        "textAlign": ["left", "left", "center"],
-        "my": [0, 0, "xxs"],
-        "px": [0, 0, "md"],
-        "display": ["inline", "inline", "block"]
-      },
-
-      "a": {
-        "fontSize": ["base", "base", "lg"],
-        "textDecoration": "underline"
-      },
-
-
-      "content": {
-        "display": "flex",
-        "alignItems": "flex-end",
 
-        "modules": {
-          "width": ["100%", "50%", "50%"],
-          "mb": "lg"
-        },
+  
+  "accordion": {
+    "button": {
+      "padding": "sm",
+      "bg": ["grey-05", "transparent"],
+      "fontFamily": "base",
+      "fontSize": "base",
+      "fontWeight": "bold",
+      "color": "textColor",
+      "alignItems": "center",
+      "borderWidth": "0 0 1px 0",
+      "borderStyle": "solid",
```

Comment:
Is this specifically overriding something? If not I think it would be better to write `borderRightWidth` (or whatever it actually is - I have no idea what order they're in) to make it clearer which border it actually is

Reply:
oh it's a button, ignore this